spec/parsers/rw2_parser_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+
+describe FormatParser::RW2Parser do
+  context 'when likely_match? is called' do
+    %w[rw2 RW2 raw RAW rwl RWL].each do |extension|
+      context "with a file with a .#{extension} extension" do
+        it "should return true" do
+          expect(subject.likely_match?("foo.#{extension}")).to be_truthy
+        end
+      end
+    end
+
+    ['', 'foo', 'rw2', 'foorw2', 'foo.rw', 'foo.rw2.bar'].each do |filename|
+      context "with a file named #{filename}" do
+        it 'should return false' do
+          expect(subject.likely_match?(filename)).to be_falsey
```

Comment:
Just bringing this up: `be_falsey` is not really `be(false)`; the latter implies it, but it's not going both ways.